double_number.rb

```diff
@@ -0,0 +1,19 @@
+class DoubleElement
+  def self.double_numbers(input)
+    input.map! { |input_element|  /\A[-+]?\d+\z/ === input_element ? input_element.to_i*2:  /\d+[,.]\d+/ === input_element ? input_element.to_f * 2 : input_element}

```

Comment:
The approach is good but can you do it without regular expression? I think you can create another function with simple approach.